lib/Service/ColumnTypes/TextLinkBusiness.php

```diff
@@ -16,10 +16,26 @@ public function parseValue(string $value, ?Column $column = null): string {
 			]));
 		}
 
+		// if is json
+		/** @noinspection PhpComposerExtensionStubsInspection */
```

Comment:
I think you could also just add `"ext-json": "*"` as a dependency to composer.json to generally make phpstorm aware of that. We require the json extension anyways for Nextcloud server